### DISCUSSION TITLE
ip6.arpa PTR queries still sent upstream even if "Disable IPv6" is checked

### DISCUSSION THREAD

**Discussion by timkgh** [AdguardTeam/AdGuardHome#3626] [Q&A]
### Issue Details

Disable IPv6 is checked. AAAA queries are blocked as they should, but I still see PTR queries for `*.ip6.arpa` sent to the upstream private DNS server.

I think these should also be blocked.

**Comment by ainar-g** [comment]
Hello.  The previously-poorly-named “Disable resolving of IPv6 addresses” option only disables the resolving of `AAAA` queries.  And I'm not sure if we want it to do more than that (@ameshkov?).

You can block all such PTR queries by a simple `||ip6.arpa` custom rule.
